docs-src/concepts/what-is-the-temporal-web-ui.md

```diff
@@ -104,8 +106,22 @@ It works only if a Worker is running and available to return the stack trace.
 
 Lists all Queries sent to the Workflow Execution.
 
+### Schedules
+
+On Temporal Cloud and self-hosted Temporal Cluster Web UI, the Schedules page lists all the [Schedules](/workflows#schedule) created on the selected Namespace.
+
+Click a Schedule to see details, such as configured frequency, start and end times, and recent and upcoming runs.
+
 ### Settings
 
+On Temporal Cloud, the Settings option is visible only to [Global Admins](/cloud/#account-level-roles).
```

Comment:
```suggestion
On Temporal Cloud, the Settings page is visible only to [Global Admins](/cloud/#account-level-roles).
```

Reply:
Not just the page but the link in the left nav vanishes too if a user isn't a Global Admin.

Reply:
Updated this to
`On Temporal Cloud, **Settings** is visible only to [Global Admins](/cloud/#account-level-roles).`

Thanks @djmagee !